tests/test_wps_generate_climos.py

```diff
@@ -73,6 +116,7 @@ def test_wps_gen_climos_opendap(netcdf, kwargs):
 def test_format_resolutions(resolutions, expected):
     gc = GenerateClimos()
     output = gc.format_resolutions(resolutions)
+    print(output)
```

Comment:
This probably got left in here by mistake, but we don't want `prints` in the tests.